_generated/CargoAuthenticateV0.versionmap.txt

```diff
@@ -0,0 +1,2 @@
+Default|0.229.5
+Node20_229_1|0.229.6
```

Comment:
is this supposed to match the task.json updates ? These do no seem to match ?

Reply:
Yes we'll need to update these patch versions when we update the tasks. Here's a doc mentioning how we bump up these versions [doc](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md). Looks that we'll need to update these versions again to 2.230.0 etc since it's entering a new sprint soon.